docs/test-vectors/Entity.json

```diff
@@ -0,0 +1,93 @@
+{
+  "good": [
+    {
+      "@context": [
+        "https://w3id.org/traceability/v1"
+      ],
+      "type": "Organization",
+      "name": "Buckridge Group",
+      "description": "Enhanced human-resource benchmark",
+      "address": {
+        "type": "PostalAddress",
+        "streetAddress": "2711 Nikki Keys",
+        "addressLocality": "South Ryannville",
+        "addressRegion": "Alaska",
+        "postalCode": "85791",
+        "addressCountry": "Poland"
+      },
+      "email": "Raymundo_Hodkiewicz@yahoo.com",
```

Comment:
let's use example.org and/or example.com for all email addresses

Reply:
Okay, changes made

Reply:
see also https://github.com/w3c/did-core/pull/489#issuecomment-744544672